lib/pluginmanager/update.rb

```diff
@@ -62,6 +63,20 @@ def update_gems!
     display_bundler_output(output)
   end
 
+  # validate if there is any major version update so then we can ask the user if he is
+  # sure to update or not.
+  def validate_major_version(plugin)
+    require "gems"
+    latest_version  = Gems.versions(plugin)[0]['number'].split(".")
```

Comment:
can we add some exception handling here for when a plugin is not found in rubygems?

It doesn't make sense to, but some may run this on a local gem that is not on rubygems.